samples/bluetooth/fast_pair/locator_tag/sysbuild/mcuboot/prj.conf

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_PM=n
```

Comment:
as I understand it, sysbuild automatically enables the proper (direct-xip mode of the bootloader or any other in case of non-nrf52 platforms)?

Reply:
Yes, MCUboot mode is automatically propagated from the sysbuild option to bootloader and appliaction image (see `nrf/sysbuild/CMakeLists.txt:L179-221`)